helper/ssh/provisioner.go

```diff
@@ -25,7 +28,7 @@ const (
 
 	// DefaultScriptPath is used as the path to copy the file to
 	// for remote execution if not provided otherwise.
-	DefaultScriptPath = "/tmp/script.sh"
+	DefaultScriptPath = "/tmp/script_%RAND%.sh"
```

Comment:
This is kind of weird but I've decided not to document this since it is a very advanced use case. And it looks like most people don't use this arg anyways. I'd rather not promise any backwards compat on the syntax of this.